packages/json-hash/src/algorthms.js

```diff
@@ -0,0 +1,18 @@
+export const algos = {
+  "SHA-1": "sha1",
+  "SHA-256": "sha256",
+  "SHA-384": "sha384",
+  "SHA-512": "sha512"
+};
+
+export const getNodeJSAlgoName = (algorithm) => {
+  return algos[algorithm];
+};
+
+export const isAvailableAlgo = (algorithm) => {
+  if (Object.prototype.hasOwnProperty.call(algos, algorithm)) {
```

Comment:
We can use algos[algorithm] here instead right, will be more readable

Reply:
Will do

Reply:
Done